domains/vplaza.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "Vplaza",
+  "repo": "https://github.com/YAsdgfterf/register/",
+  "owner": {
+    "username": "YAsdgfterf",
+    "email": "amongusmemes330@gmail.com"
+  },
+  "record": {
+    "A": "213.136.73.63"
```

website doesnt seem to be developer related, tos not accepted
A records should be in an array
```suggestion
    "A": ["213.136.73.63"]
```